pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -178,6 +177,11 @@ stdenv.mkDerivation {
       for i in ${builtins.toString perlLibs}; do
         gitperllib=$gitperllib:$i/lib/perl5/site_perl
       done
+
+      ln -s "$out/share/git/contrib/credential/netrc/git-credential-netrc" $out/bin/
+      wrapProgram $out/bin/git-credential-netrc \
+                  --set PERL5LIB "$gitperllib"
```

Comment:
```
makeWrapper "$out/share/git/contrib/credential/netrc/git-credential-netrc" $out/bin/git-credential-netrc \
  --set PERL5LIB "$gitperllib"
```
could replace both ```ln``` and ```wrapProgram``` (but I have not tested)

Reply:
Thanks!

https://nixos.org/nixpkgs/manual/#ssec-stdenv-functions says "wrapProgram is a convenience function you probably want to use most of the time." I'm new to nixpkgs - could you explain why you'd prefer makeWrapper here?